examples/layers/preprocessing/mix_up_demo.py

```diff
@@ -0,0 +1,46 @@
+"""mix_up_example.py shows how to use the MixUp preprocessing layer to 
```

Comment:
Same for this module, please check all the comments in cut_mix_demo.py.

Reply:
I think this is now fixed - thanks